MSBLOC.Core/Services/CheckRunSubmitter.cs

```diff
@@ -37,7 +37,7 @@ public class CheckRunSubmitter : ICheckRunSubmitter
                 };
 
                 return newCheckRunAnnotation;
-            }).ToList();
+            }).Take(50).ToList();
```

Comment:
OctokitGraphql made me realize we are limited to 50 annotations

Reply:
Okay, we are limited to creating 50 annotations per GitHub Api request. We can create more, just have to use the "Update Check Run" request. #43